src/components/pages/daysoff.js

```diff
@@ -63,6 +64,27 @@ export default function Daysoff() {
             showToast(getText('daysoff.error.action'));
         }
     };
+
+    const numberOfSelectedFilter = () => {
```

Comment:
* This should be named `countActiveFilters`
* This should be placed in the `filter` context
* There should be a `activeFilterCount` state in the `filter` context that is updated any time filter data is updated